translators/test/snapshot/ChainDefFields_NonDependentRecordModule.lean

```diff
@@ -22,4 +21,4 @@ structure Record5 where
     f5 : Nat
 
 open Record1 Record2 Record3 Record4 Record5
-example : Record5 := Const5 1
\ No newline at end of file
+example : Record5 := Const51
```

Comment:
Is this correct? Looks like a bug!

Reply:
It sure does. Well spotted. I'll track it down.

Reply:
Fixed now. Affected 5 test cases!